### PR TITLE
feat: show basic info start-from-init

### LINKED ISSUE #6113: Show some basic information when running `start-from-init`
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

When starting ZITADEL following the [docker compose guide](https://zitadel.com/docs/self-hosting/deploy/compose), I'm sometimes not sure if the right version was applied (uses latest). Also when changing external domains etc, I might not be able to access the Console as described in our docs.
It would be helpful to post some basic information to stdout so the user gets some basic properties and urls.

### Describe your ideal solution

It would be great to get some basic information and urls at startup, so I don't have to figure them out on my own. These could include: 

- Version
- url to console
- url to check healthz
- Security notice (eg, using tlsMode disabled)

### Version

2.29.0

### Environment

Self-hosted

### Additional Context

When using ZITADEL in combination with cockroachdb, then I get the crdb output at startup, which shows me some basic information and warnings that seem helpful.

### PR DESCRIPTION
**Pull request by doncicuto**
As @mffap states: 

> When starting ZITADEL [...] I'm sometimes not sure if the right version was applied (uses latest). Also when changing external domains etc, I might not be able to access the Console as described in our docs. It would be helpful to post some basic information to stdout so the user gets some basic properties and urls.

This PRs proposes the following information:

image

Closes #6113 

### Definition of Ready

- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [X] No debug or dead code
- [X] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [X] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Review by muhlemmer — CHANGES_REQUESTED** [review]
thanks for this great addition! just a few comments/questions.

File: .goreleaser.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ builds:
     goarch:
       - amd64
       - arm64
-    ldflags: -s -w -X github.com/zitadel/zitadel/cmd/build.version={{.Version}} -X github.com/zitadel/zitadel/cmd/build.commit={{.Commit}} -X github.com/zitadel/zitadel/cmd/build.date={{.Date}}
```
[the remark this answers is not in the document]
Comment (doncicuto) [reply .goreleaser.yaml:36]: My bad! Sorry, that was unintended. I'll fix that 😄

File: go.mod [hunk 2]
```diff
@@ -87,6 +87,8 @@ require (
 
 require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/resourcemapping v0.37.0 // indirect
+	github.com/common-nighthawk/go-figure v0.0.0-20210622060536-734e95fb86be // indirect
```
[the remark this answers is not in the document]
Comment (doncicuto) [reply go.mod]: Done!

File: cmd/start/start_from_init.go [hunk 3]
```diff
@@ -37,6 +42,8 @@ Requirements:
 
 			startConfig := MustNewConfig(viper.GetViper())
 
+			showBasicInformation(startConfig)
```
[the remark this answers is not in the document]
Comment (doncicuto) [reply cmd/start/start_from_init.go]: Done!

**Comment by doncicuto** [comment]
Thanks @muhlemmer for your time reviewing my code!

**Review by muhlemmer — APPROVED** [review]

**Pull request merged**